docker-compose.postgres.yml

```diff
@@ -6,26 +6,29 @@ services:
 
   vdi:
     depends_on:
-      - edapostgres
+      edapostgres:
+        condition: service_healthy
 
   eda:
     depends_on:
-      - edapostgres
+      edapostgres:
+        condition: service_healthy
 
   wdk:
     depends_on:
-      - edapostgres
+      edapostgres:
+        condition: service_healthy
 
   edapostgres:
     build:
       context: ./postgres-build
     environment:
-      POSTGRES_USER: ${USER_DB_USER:?}
-      POSTGRES_PASSWORD: ${USER_DB_PASS:?}
+      POSTGRES_USER: ${POSTGRES_ROOT_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_ROOT_PASSWORD}
       POSTGRES_DB: eda
       PGPORT: 5434
       VDI_SCHEMA_SUFFIX: ${VDI_SCHEMA_SUFFIX}
-      VDI_DB_PASSWD: ${USER_DB_PASS:?}
+      VDI_DB_PASSWD: ${DB_CONNECTION_PASS_MICROBIOME}
```

Comment:
Curious why this is a different password than above?  In the init scripts do we restrict access to this schema in a different way than we do the eda data?

Reply:
Yeah, we support having a different password for the VDI schemas since VDI logs in as the "vd_control_<suffix>" user. This is reflected in the init scripts as well in the for of creating a user.

Other EDA access is just using the postgres root for now, though we would probably want to have either the init scripts or the DBA create separate users for each app for security purposes.